shell/platform/darwin/macos/framework/Source/FlutterExternalTextureGL.mm

```diff
@@ -0,0 +1,91 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterExternalTextureGL.h"
+
+#import <AppKit/AppKit.h>
+#import <CoreVideo/CoreVideo.h>
+#import <OpenGL/gl.h>
+
+static void OnGLTextureRelease(CVPixelBufferRef pixelBuffer) {
```

Comment:
The `user_data` you're passing is a `CVOpenGLTextureCacheRelease`. Why is this using `CVPixelBufferRef` as the type and `CVPixelBufferRelease`? In practice it presumably works, since most of the specialized releases are just calling CFRelease, but this seems very confusing.

Reply:
Yes, need to change to CVOpenGLTextureRelease.